### ISSUE TITLE
Issue parsing databags

### ISSUE THREAD

**Issue by ewhauser** [bmarini/health_inspector#12]
I got the following error when trying to print the differences in data bags:

```
/Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:157:in `indent': undefined method `prepend' for "\e[93mgecos : \e[0m":String (NoMethodError)
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:124:in `print_key'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:111:in `print_failures_from_hash'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:110:in `each'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:110:in `print_failures_from_hash'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:102:in `print_failures'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:99:in `each'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:99:in `print_failures'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:39:in `run'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:33:in `each'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:33:in `run'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/checklists/base.rb:23:in `run'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/inspector.rb:15:in `inspect'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/inspector.rb:4:in `inspect'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/lib/health_inspector/cli.rb:15:in `inspect'
    from /Library/Ruby/Gems/1.8/gems/thor-0.16.0/lib/thor/task.rb:27:in `send'
    from /Library/Ruby/Gems/1.8/gems/thor-0.16.0/lib/thor/task.rb:27:in `run'
    from /Library/Ruby/Gems/1.8/gems/thor-0.16.0/lib/thor/invocation.rb:120:in `invoke_task'
    from /Library/Ruby/Gems/1.8/gems/thor-0.16.0/lib/thor.rb:275:in `dispatch'
    from /Library/Ruby/Gems/1.8/gems/thor-0.16.0/lib/thor/base.rb:425:in `start'
    from /Library/Ruby/Gems/1.8/gems/health_inspector-0.2.0/bin/health_inspector:3
    from /usr/bin/health_inspector:23:in `load'
    from /usr/bin/health_inspector:23
```

**Comment by adamsinnett** [comment]
This is an issue with a method missing in older ruby's. We experienced it as well and I've submitted a pull request that will fix it.

**Comment by bmarini** [comment]
I just pushed a fix out for that in 0.2.1 Now get on ruby 1.9 already ;)
